crates/notary/server/fixture/tee/Dockerfile

```diff
@@ -0,0 +1,25 @@
+#tlsnotary server for testing <> gramine sgx (gramine1.7, g++13, libiomp off :()
```

Comment:
any reason ure putting these tee files in `fixture`? asking this cause `fixture` here is mainly for mock data for testing or local trial purposes

Reply:
`crates/notary/server/tee`